lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -134,18 +163,5 @@ $notification-item-height: 72px;
 
     #{$mat-menu-content} {
         padding: 0;
-
-        #{$mat-list} {
```

Comment:
$mat-list - is that selector used anywhere or can we remove that from selectors list because I can't see usages?

Reply:
I don't see usages as well, deleted